src/test/java/com/marklogic/mule/extension/connector/internal/operation/MarkLogicOperationTest.java

```diff
@@ -88,7 +87,7 @@ public void setUp() {
         configuration.setServerTransformParams(TRANSFORM_PARAMS.toString());
         configuration.setSecondsBeforeFlush(FLUSH_SECONDS);
         configuration.setJobName(JOB_NAME);
-        connection = new MarkLogicConnection(prop.getProperty("config.hostName"), PORT, DATABASE_NAME, prop.getProperty("config.username"), prop.getProperty("config.password"), AuthenticationType.digest, MarkLogicConnectionType.DIRECT, null, null, CONNECTION_ID);
+        connection = new MarkLogicConnection(prop.getProperty("config.hostName"), PORT, null, prop.getProperty("config.username"), prop.getProperty("config.password"), AuthenticationType.digest, MarkLogicConnectionType.DIRECT, null, null, CONNECTION_ID);
```

Comment:
Let's leave the database param as null for now, like you have it. I think I added that to specifically test the lines of code covering a named database, as opposed leaving them uncovered and falling back to an implied one through the MarkLogic app server config. But as long as we get the code coverage elsewhere, I'm fine with this. Most MarkLogic users won't use it with a named DB anyway, and rely on the app server port mapping like you suspect.